karma.conf.js

```diff
@@ -24,6 +24,9 @@ module.exports = function(config) {
     },
 
     files: [
+      { pattern: 'node_modules/expect.js/index.js', watched: false },
+      { pattern: 'node_modules/moment/min/moment.min.js', watched: false },
+      { pattern: 'dist/moment-range.js', watched: false },
```

Comment:
I didn't like to add the dependencies, but the test does not work without this. 
Of course, I could define a second karma configuration especially for the browser test, if you think it's worth it.